gnomADc.pm

```diff
@@ -159,16 +130,24 @@ sub get_header_info {
   my $self = shift;
 
   my $prefix = $self->{prefix};
+  my $header = $self->{file_column} if defined($self->{file_column});
   my %header_info;
+  
 
-  for (qw(mean median)) {
-    $header_info{ join('_', $prefix, $_, 'cov') } = "$_ coverage";
+  if (defined($header == 14 )){
```

Comment:
v2 exomes header is returning total_DP because `defined($header == 14 )` always returns true. You should change it to: `if (defined $header && $header == 14)`